lib/node_modules/@stdlib/blas/base/snrm2/README.md

```diff
@@ -86,18 +84,15 @@ Note that indexing is relative to the first index. To introduce an offset, use [
 
 ```javascript
 var Float32Array = require( '@stdlib/array/float32' );
-var floor = require( '@stdlib/math/base/special/floor' );
 
 var x0 = new Float32Array( [ 2.0, 1.0, 2.0, -2.0, -2.0, 2.0, 3.0, 4.0 ] );
 var x1 = new Float32Array( x0.buffer, x0.BYTES_PER_ELEMENT*1 ); // start at 2nd element
 
-var N = floor( x0.length / 2 );
-
-var z = snrm2( N, x1, 2 );
+var z = snrm2( 4, x1, 2 );
 // returns 5.0
 ```
 
-If either `N` or `stride` is less than or equal to `0`, the function returns `0`.
+If either `N` or stride is less than or equal to `0`, the function returns `0`.
```

Comment:
```suggestion
If either `N` or `stride` is less than or equal to `0`, the function returns `0`.
```

Backticks are necessary here, otherwise it doesn't read well.